### ISSUE TITLE
OCR is broken for Yostar KR client

### ISSUE THREAD

**Issue by 178619** [MaaAssistantArknights/MaaAssistantArknights#3486]
### Before raising the issue...

- [X] I understand that Issues are for feedback and problem solving, not trolling the comments section, and will provide as much information as possible to help solve the problem.
- [X] I filled in a short, clear title so that developers could quickly identify the general problem when going through the issue list. Instead of "Some suggestions", "Stuck", etc.
- [X] I have checked the latest beta update and there is no mention of the bug being fixed.

### Description

```
[2023-01-29 14:26:32.129][TRC][Px2b68][Tx2e10] OcrPack::recognize | raw: [{ 꿀깰놓옐킥놓옐놓日緣建離建日建誤틔誤: [ 0, 0, 230, 100 ], score: 0.054207 }]
[2023-01-29 14:26:32.881][TRC][Px2b68][Tx2e10] OcrPack::recognize | raw: [{ 誤옐꿀듐建옐깰혜建누指놓옐離答指答建틔誤: [ 205, 9, 91, 26 ], score: 0.100748 }]
[2023-01-29 14:26:33.826][TRC][Px2b68][Tx2e10] OcrPack::recognize | raw: [{ 建故B建離놓指놓옐建離옐建勸틔Ⅷ誤: [ 85, 26, 92, 29 ], score: 0.089344 }]

[2023-01-29 14:30:35.422][TRC][Px2b68][Tx2e10] OcrPack::recognize | raw: [{ ş옐離絹놓쇳ş혔옐建옐離혔建혔듐밥듐離誤離建誤틔Ⅷ誤Ⅷ建: [ 123, 8, 176, 53 ], score: 0.071503 }]
[2023-01-29 14:30:36.695][TRC][Px2b68][Tx2e10] OcrPack::recognize | raw: [{ 建옐놓建離깰指옐.建옐離놓建틔誤: [ 23, 28, 146, 55 ], score: 0.110252 }]
[2023-01-29 14:30:36.719][TRC][Px2b68][Tx2e10] OcrPack::recognize | raw: [{ ş옐離깰놓ş혔離자建혔建혔듐뒷혔離혔建誤Ⅷ誤Ⅷ建: [ 121, 2, 178, 61 ], score: 0.102522 }]
```
Returns random characters with a low score repeatedly.

### Log files

[asst.log](https://github.com/MaaAssistantArknights/MaaAssistantArknights/files/10529033/asst.log)

**Comment by MistEO** [comment]
Is this an accidental or inevitable issue?  Can a re-download fix it?  Looks like the resource is corrupted.

**Comment by 178619** [comment]
Redownloading didn't work. Tried manually replacing `resource\global\YoStarKR\resource\PaddleOCR\rec` with files at Github but it didn't work either.

Also tried to replace OCR files with resources from v4.7.3. It worked as if the PaddleOCR folder didn't exist. (Returns Chinese characters that look similar to Korean characters in images)

**Comment by MistEO** [comment]
Maybe I corrupted the resource while converting or copying, and I'll try converting it again later.

If you want to try to fix it, you can convert the resources in v4.7.3 using Paddle2Onnx (convert paddle resources to onnx resources). There is no need to use ArkTrainingData, it is used to train paddle resources.
